transmission-visualizer/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+import org.jetbrains.intellij.platform.gradle.TestFrameworkType
+
+plugins {
+    alias(libs.plugins.jetbrainsKotlinJvm)
+    id("org.jetbrains.intellij.platform") version "2.6.0"
```

Comment:
Before merging this, it would be nice to test publishing it locally and see if it works.